### DISCUSSION TITLE
Question : how are stored writers, tags, etc...

### DISCUSSION THREAD

**Discussion by racine-p-a** [anansi-project/comicinfo#45] [Q&A] [answered]
Hello (and thanks for your work).

  I am currently working on an interface that could create `ComicInfo.xml` files while respecting your norm. One of my problems is : how are we suppose to store multiple tags or writers (or else...) ?
Let's take writers as an example. I want to add multiple ones on a single comic but, as stated in your `xsd` file [here](https://github.com/anansi-project/comicinfo/blob/main/drafts/v2.1/ComicInfo.xsd), only one tag of `Writer` is authorized.
My guess is that I am supposed to place them in a single string with a separator. In this case, what separator shall I use ?
```xml
  <Writer>Author1, Author2</Writer>
OR
<Writer>Author1| Author2</Writer>
OR
<Writer>Author1,;Author2</Writer>
OR
<Writer>Author1_separator-dependant-of-application_Author2</Writer>
```

Thank you for your answer(s) and sorry for my broken english.

**Comment by majora2007** [comment, marked as answer]
They are comma separated.

**Comment by gotson** [comment]
You can check the schema documentation https://anansi-project.github.io/docs/comicinfo/documentation

Accepted usage is as majora said, comma separated.

**Reply by baodrate** [reply]
I assume the convention is just to remove commas from names that have them? Or is there another way to retain the comma?

**Reply by majora2007** [reply]
I haven't seen any names with commas yet. If you have last, first, you would reorder to first last.

**Reply by baodrate** [reply]
There are cases on ComicVine (the main source of my app's metadata). e.g. ["Mike Deodato, Jr."](https://comicvine.gamespot.com/mike-deodato-jr/4040-21341/). Not a case of "Last, First" reordering.

My app is producing the comicinfo.xml when it tags comics. But that means I have to strip the commas if I want to match the data back up with the database (e.g. to display more info about the specific creator), especially since you can't extend an xml schema to add an `Id` attribute.

I could always strip the commas from the data coming from CV, or do it in the query, but ideally I could avoid doing either

**Reply by lordwelch** [reply]
ComicTagger since 2012 has just removed them https://github.com/comictagger/comictagger/commit/ef599b9403ff282aa3e46fec54d184e253720e1b
ComicRack, where this standard came from, doesn't let you use commas at all. If you paste his name in you get two people "Mike Deodato" and "Jr", this standard has no way to support using a comma in a credit in any capacity except to separate different names. ComicRack also does not support other characters as delimiters.

**Reply by baodrate** [reply]
fair enough, thanks

**Comment by racine-p-a** [comment]
Thank you for your quick answers.
